allennlp/data/token_indexers/single_id_token_indexer.py

```diff
@@ -16,7 +19,10 @@ class SingleIdTokenIndexer(TokenIndexer):
     # Parameters
 
     namespace : `str`, optional (default=`tokens`)
-        We will use this namespace in the :class:`Vocabulary` to map strings to indices.
+        We will use this namespace in the :class:`Vocabulary` to map strings to indices.  If you
+        explicitly pass in `None` here, we will skip indexing and vocabulary lookups.  This means
```

Comment:
The type annotation for `namespace` should be `Optional[str]` then.

Reply:
Done.